debuglogger/level.go

```diff
@@ -67,7 +44,7 @@ func (level LogLevel) String() string {
 	return "UNKNOWN"
 }
 
-// Invalid returns true if the log level is invalid
-func (level LogLevel) Invalid() bool {
-	return level < LogLevelNoLog || level > LogLevelTrace
+// Valid returns true if the log level is invalid
```

Comment:
Typo:  "Valid returns true if the log level is invalid" --> "Valid returns true if the log level is valid"